src/commands/supported-tokens.ts

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable @typescript-eslint/no-base-to-string */
-import { type Addressable, Contract } from 'ethers'
+import { type Addressable, type JsonRpcApiProvider, Contract } from 'ethers'
 import type { TypedContract } from 'ethers-abitype'
 
+import { chunk } from 'lodash-es'
```

Comment:
Could we please not pull whole `lodash-es` new dependency only for this small util? It may be easy enough to code it inline or write a function here for this